### PR TITLE
Fix logo, slider upload and display problems

### PR DESCRIPTION
**Pull request by vamcart**
Fix for issue #17

### CODE REVIEW

File: theme/src/components/footer.js [hunk 1]
```diff
@@ -1,135 +1,136 @@
...
+									items={themeSettings.footer_menu_2_items}
+								/>
+							</div>
+						</div>
+					</div>
+				</footer>
+			</section>
+		);
+	}
+}
```
Comment (Bogdan-p) [inline theme/src/components/footer.js:136]: Why is all this file marked as changed?

Comment (vamcart) [reply theme/src/components/footer.js:136]: I don't know. It's github.
In footer.js changed 1 line:

		const footerLogoUrl =
			themeSettings.footer_logo_url && themeSettings.footer_logo_url.length > 0
				? storeSettings.assetsBaseURL + '/assets/images/' + themeSettings.footer_logo_url
				: settings.logo;

Comment (Bogdan-p) [reply theme/src/components/footer.js:136]: Yes, indeed. There's an option to ignore whitespaces in **Diff settings**.
Wouldn't it be more clear with template literals? Something like this:
`${storeSettings.assetsBaseURL}/assets/images/${themeSettings.footer_logo_url}`

Comment (vamcart) [reply theme/src/components/footer.js:136]: Maybe. But i'm waiting for approve all of 3 PR. We can change this string in the further commits, when this PR will be merged to the main code.

I test it on my local machine with ubuntu and i'm happy now with cezerin2 separated repos.
Now i have all working as i want (product images in admin and storefron, slider images in admin and storfront, logo in admin and storefront).

But you need to test all of 3 Pull Requests to take this work as expected.
cezerin2, cezerin2-admin, cezerin2-store changes.

Logos, sliders, products images uploading and displaying correctly at admin and the store now.

Comment (Bogdan-p) [reply theme/src/components/footer.js:136]: ok, I'm going to test these also, as i was just struggling to fix the assets on store/admin! Good job!

Comment (vamcart) [reply theme/src/components/footer.js:136]: All changes made based on product images that displayed at storefront. As you can see storefront and product images at the storefront - all url came from api localhost:3001/content/images/...

On this base we made slider images and logo images at storefront and admin.

For me all works fine (installation from source code, not docker installation)

File: theme/src/components/homeSlider.js [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
+				style={{ color: themeSettings.home_slider_color || '#fff' }}
+			>
+				<div className="caption-title">{item.title}</div>
+				<div className="caption-description">{item.description}</div>
+			</div>
+		</NavLink>
+	</div>
+);
+
+const HomeSlider = ({ images, settings }) => {
```
Comment (Bogdan-p) [inline theme/src/components/homeSlider.js]: settings is not used, i think it can be removed

File: theme/src/components/homeSlider.js [hunk 3]
```diff
@@ -55,6 +56,7 @@ const HomeSlider = ({ images }) => {
 
 HomeSlider.propTypes = {
 	images: PropTypes.arrayOf(PropTypes.shape({}))
+	//settings: PropTypes.shape({}).isRequired
```
Comment (Bogdan-p) [inline theme/src/components/homeSlider.js]: The comment can be removed

File: theme/src/containers/index.js [hunk 4]
```diff
@@ -1,67 +1,67 @@
...
+		<Fragment>
+			<MetaTags
+				title={pageDetails.meta_title}
+				description={pageDetails.meta_description}
+				canonicalUrl={pageDetails.url}
+				ogTitle={pageDetails.meta_title}
+				ogDescription={pageDetails.meta_description}
+			/>
+
+			<HomeSlider images={themeSettings.home_slider} settings={settings} />
```
Comment (Bogdan-p) [inline theme/src/containers/index.js]: The `settings` prop can be removed from here too

File: theme/src/components/homeSlider.js (hunk 2 above)
Comment (vamcart) [reply theme/src/components/homeSlider.js]: Thanks, removed

File: theme/src/components/homeSlider.js (hunk 3 above)
Comment (vamcart) [reply theme/src/components/homeSlider.js]: Thanks, removed

File: theme/src/containers/index.js (hunk 4 above)
Comment (vamcart) [reply theme/src/containers/index.js]: Thanks, removed

**Review by ngoyal16 — COMMENTED** [review]
config store.production.js should have env based variables. so that easy for docker too

**Review by ngoyal16 — COMMENTED** [review]
this setting also needs to be removed.. because API server should send the assets full path

**Pull request merged**